web/webkit/src/main/scala/net/liftweb/http/Req.scala

```diff
@@ -284,14 +284,14 @@ FileParamHolder(name, mimeType, fileName)
  * @param localFile The local copy of the uploaded file
  */
 class OnDiskFileParamHolder(override val name: String, override val mimeType: String,
-                            override val fileName: String, val localFile: File) extends
+                            override val fileName: String, val localFile: Path) extends
```

Comment:
Heh, so the reason `FileInputStream` and `FileOutputStream` are “considered harmful” in the Cloudbees post is because they use finalizers. But `OnDiskFileParamHolder` also has a finalizer. I think this is probably still a win, but worth mentioning.